src/frontend/src/common.ts

```diff
@@ -1,4 +1,5 @@
 
 export const isProduction = process.env.NODE_ENV === "production";
 
-export const baseUrl = isProduction ? window.location.origin : "http://localhost:8000";
+// export const baseUrl = isProduction ? window.location.origin : "http://localhost:8000";
```

Comment:
Sehr cool! Kannst du noch in einem Kommentar dazuschreiben, wieso es noch in einem Kommentar da bleibt? Dann kann man es in der Zukunft sorgenlos entfernen, wenn der Grund entfällt.